src/GravitationalClustering.js

```diff
@@ -65,6 +48,7 @@ export default class GravitationalClustering {
         this.eps = Number.MAX_VALUE;
         this.engine = Random.engines.mt19937();
         this.engine.seed(this.seed);
```

Comment:
I used this for reproducibility of the experiments (if I use Math.random I cannot seed it), most of the functions on sklearn use seeds instead of providing a random function generator, also I still have the same problem as before, I have to save the function and load it again(this is why I used [Merssene-twister](https://en.wikipedia.org/wiki/Mersenne_Twister), because is the most used on this kind of experiments).